### ISSUE TITLE
How to resume the training

### ISSUE THREAD

**Issue by zjc24** [yjsunnn/DLoRAL#18]
Thank you for your great work! I would like to know, if my training gets interrupted, how can I resume the training? I saw that in the Generator there is a 'load_ckpt_from_state_dict' method, but it only loads the model weights, without loading the optimizer, learning rate scheduler, global_step, and other parameters.
